internal/website/content/ref/docstore.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "docstore"
+date: 2019-08-14T00:00:00-08:00
+aliases:
+- /pages/docstore/
+---
+
+The docstore package provides an abstraction layer over common document stores
+like Google Cloud Firestore, Amazon DynamoDB, and MongoDB.
+
+<!--more-->
+
+[How-to guide]({{< ref "/howto/docstore/_index.md" >}})<br>
+[Top-level package documentation](https://godoc.org/gocloud.dev/docstore)
+
+## Supported Services
+
+* [GCP Firestore](https://godoc.org/gocloud.dev/docstore/gcpfirestore)
+* [AWS DynamoDB](https://godoc.org/gocloud.dev/docstore/awsdynamodb)
+* [Azure blob](https://godoc.org/gocloud.dev/docstore/azureblob)
```

Comment:
Remove this line as Cosmos DB is listed under MongoDB.

Reply:
Done.